fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -236,6 +237,21 @@ public void testHeadBasicContainer() throws IOException {
     }
 
     @Test
+    public void testHeadDefaultContentType() throws IOException {
+        final String id = getRandomUniqueId();
+        createObjectAndClose(id);
+        addMixin(id, CONTAINER.getURI());
+
+        final HttpHead headObjMethod = headObjMethod(id);
+        try (final CloseableHttpResponse response = execute(headObjMethod)) {
+            final Collection<String> contentType = getHeader(response, CONTENT_TYPE);
+            assertTrue(contentType.size() == 1);
+            assertTrue("Didn't find LDP valid content-type header: " + contentType, !contentType.iterator().next()
```

Comment:
I'm having a hard time getting jersey to return a deterministic result when no accept header is specified.  How about I add additional tests that verify that the using an appropriate accept header gives the correct content-type in the response?.